src/js/components/Page/stories/CustomThemed/LeftColumn.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import {
+  Page,
+  Box,
+  Header,
+  Heading,
+  Paragraph,
+  Grid,
+  Card,
+  PageSection,
+  Grommet,
+} from 'grommet';
+
+const customTheme = {
+  page: {
+    customKind: {
+      alignSelf: 'start',
+      width: {
+        min: '200px',
+        max: '500px',
+      },
+      pad: {
+        small: 'medium',
+        medium: 'medium',
+        large: 'medium',
+      },
+      margin: { vertical: 'small', horizontal: 'small' },
+    },
+  },
+};
+
+export const LeftColumn = () => (
+  <Grommet theme={customTheme}>
+    <Box background="#35bd55">
+      <Heading>Global Header</Heading>
+    </Box>
+    <Page kind="customKind">
+      <PageSection>
+        <Header>
+          <Heading>Heading</Heading>
```

Comment:
I'm finding these sections really large and it's making it hard for me to understand the page structure
```suggestion
          <Heading size="small">Heading</Heading>
```